doc/sphinx/language/core/definitions.rst

```diff
@@ -121,8 +121,8 @@ Assertions and proofs
 ---------------------
 
 An assertion states a proposition (or a type) of which the proof (or an
-inhabitant of the type) is interactively built using tactics. The interactive
-proof mode is described in Chapter :ref:`proofhandling` and the tactics in
+inhabitant of the type) is interactively built using tactics.
+Proof mode is described in Chapter :ref:`proofhandling` and tactics in
 Chapter :ref:`Tactics`. The basic assertion command is:
```

Comment:
What about linking to the top-level chapter ("Basic proof writing") instead? This would be more resilient to the already done and planned splitting of the Tactics chapter.